manifests/init.pp

```diff
@@ -119,7 +119,10 @@
       icvpn_as => $mesh_as;
     }
   }
- 
+  
+  ffnord::resources::ffnord::field {
+    "MESH_CODE": value => "${mesh_code}";
```

Comment:
I think all your use cases could be handled without this variable and instead use find/sed.